src/scripts/fix-program-events-user/.eslintrc.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
@tokland Awesome! Don't know too much about this. I would like to discuss a bit about this with you on Thursday.